docs/structure-of-a-contract.rst

```diff
@@ -54,6 +54,66 @@ EVM Version
 
 The EVM version can be set with the ``evm-version`` pragma, which is documented in :ref:`evm-version`.
 
+Imports
+=======
+
+Import statements allow you to import :ref:`modules` or :ref:`interfaces` with the ``import`` or ``from ... import`` syntax.
```

Comment:
Maybe nice to specify whether we support the `from ... import ... as ...` syntax too